mesheryctl/internal/cli/root/perf/apply.go

```diff
@@ -148,54 +151,68 @@ var applyCmd = &cobra.Command{
 				profileName = response.Name
 
 				log.Debug("New profile created")
-			} else { // set profile-id from args
-				profileID = args[0]
+			} else { // set profile-name from args
+				// Merge args to get profile-name
+				profileName = strings.Join(args, "%20")
 
-				// what if user passed profile-id but didn't passed the url
-				// we fetch performance profile first
+				// search and fetch performance profile with profile-name
+				log.Debug("Fetching performance profile")
+
+				req, err = http.NewRequest("GET", mctlCfg.GetBaseMesheryURL()+"/api/user/performance/profiles?search="+profileName, nil)
```

Comment:
We should flag this for consolidation (https://docs.google.com/spreadsheets/d/1ABJCbfQRi0uN_YoP2kmHZ-lTI4S4QuvRk1_unomKNRE/edit#gid=1241442399), perhaps, by opening an issue to track the need to update this endpoint.